Components/Managers/ChiselModelManager.cs

```diff
@@ -205,6 +205,12 @@ public void Rebuild(ChiselModelComponent model)
 			}
 
 			UpdateModelFlags(model);
+
+			if(model.AutoRebuildUVs)
+			{
+				ForceUpdateDelayedUVGeneration();
+				DelayedUVGeneration(force: true);
+			}
```

Comment:
Sorry for the late comment, I was on vacation and then forgot .. 😬 
Also, it's been a while since I looked at this particular code
So, DelayedUVGeneration works on all models, so this is not the right place for this.
In UpdateModels() DelayedUVGeneration is called, after all models are done, and it should already run if ForceUpdateDelayedUVGeneration has been called before
So just calling ForceUpdateDelayedUVGeneration here should be enough